src/tseda/vpages/stats.py

```diff
@@ -79,12 +79,13 @@ def __panel__(self):
         windows = make_windows(
             self.window_size, self.datastore.tsm.ts.sequence_length
         )
-        sample_sets_list = (
-            self.datastore.individuals_table.selected_sample_set_indices()
+        samples, sample_sets_dictionary = (
```

Comment:
So apart from getting rid of get_sample_sets, the question is what sample_sets should return?

Reply:
I would say that the most natural thing would be to return them all by default, but if the number of individual samples is too high, and by that I mean too many samples in terms of performance and visualisation, then throw a warning message and limit that number? I don't know which number would be adequate, I guess it depends, but maybe we could start by like 500 samples, and see how it goes?

Reply:
@tforest Good point about sample sizes. My comment was more directed at the function signature though: does it return a (list, dict) tuple or just a dict?

Reply:
The first returned value, samples, is only used twice when the function is called while the second returned value, sample_sets (the dictionary), is always used when the function is called. I can try to rewrite the function so that it only returns the dictionary.

Reply:
@percyfal should be fixed now